src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key in robot) {
+    if (result.hasOwnProperty(robot[key])) {
```

Comment:
Replace 'result.hasOwnProperty()' with 'Object.prototype.hasOwnProperty.call()' to prevent potential issues with overridden 'hasOwnProperty'